addons/html_editor/static/tests/toolbar.test.js

```diff
@@ -347,7 +347,7 @@ test("toolbar works: can select font size", async () => {
 
     await contains(".o-we-toolbar [name='font_size_selector'].dropdown-toggle").click();
     const sizes = new Set(
-        fontSizeItems.map((item) => getFontSizeFromVar(item.variableName).toString())
+        fontSizeItems.map((item) => getFontSizeFromVar(item.variableName).toString()).sort()
     );
```

Comment:
You're sorting the array before wrapping it in a Set, but Set does not guarantee order preservation.
I suggest:
```suggestion
const sizes = [...new Set(
    fontSizeItems.map((item) => getFontSizeFromVar(item.variableName))
)].sort((a, b) => a - b).map(String);
```